### DISCUSSION TITLE
Detect black scenes doesn't seem to work for me

### DISCUSSION THREAD

**Discussion by markdelfs** [mifi/lossless-cut#1705] [Q&A]
I have a simple need--I have downloaded a Youtube video of commercials from the 80's. It is just one long video with about 100 commercials. I want to simply use Lossless-cut to go through, mark the black spots (separate each commercial) and then I can just do a batch download. 

I can do this by hand (scrub through, make a cut, scrub throug, make a cut, download clip etc.) but that takes FOREVER. 

I thought this detect black scenes would work so much faster, but, nothing happens when I run it. I have adjusted the parameters for hours and nothing changes--It just says there are no black scenes each and every time. I have even tried different videos with different commercials. 

Any tips here?

**Comment by mifi** [comment]
thanks for reporting. this is indeed a bug. I will fix it

**Reply by mifi** [reply]
meanwhile you can use an older version or wait for the next nightly build

**Reply by markdelfs** [reply]
The older version does work--it correctly goes through and marks the black spots between each commercial. But, what if I want to export the opposite of the black spots--I don't want to delete the black spots--I want to select the opposite of the black spots--then export the real commercial so I end up with 40 different commercials all neatly chopped up.

**Reply by markdelfs** [reply]
Nevermind--I figured it out. You search for black spots, then use the 'Invert all segments on Timeline' which selects the opposite of the black spots and then export. Genius software.
